sql/original_update_tests.md

```diff
@@ -0,0 +1,48 @@
+# Original Update Tests
```

Comment:
I don't see anything here about toolkit versions.  It's OK not to implement that here, but I think we at least need an idea of what that might look like before proceeding.

Reply:
I'm currently writing min_toolkit_version code, I pushed up what I had so far just in case it took longer to implement than planned. It should be up soon.

Reply:
OK I just wanted to make sure we planned for it.  No rush on pushing that.

It's great to see this progress!

Thanks.